bash.html.markdown

```diff
@@ -23,7 +23,7 @@ translators:
 ---
 
 Bash is a name of the unix shell, which was also distributed as the shell
-for the GNU operating system and as default shell on Linux and Mac OS X.
```

Comment:
I don't understand the change, isn't bash also the default shell on macOS?

Reply:
zsh has become the default shell: https://support.apple.com/en-us/HT208050

Reply:
ah okay, thank you. From Catalina onward I see.

Reply:
thank you @gfting i couldn't remember where i had seen it!